lua/telescope/_extensions/packer.lua

```diff
@@ -84,27 +84,15 @@ local plugins = function(opts)
         vim.cmd(string.format(":e %s", selection.readme))
       end)
 
-      local Job = require "plenary.job"
       local open_online = function()
         local selection = action_state.get_selected_entry()
 		actions._close(prompt_bufnr, true)
 
         local cmd = vim.fn.has "win-32" == 1 and "start" or vim.fn.has "mac" == 1 and "open" or "xdg-open"
+		local url = vim.fn.system(string.format("git -C %s ls-remote --get-url", selection.path))
```

Comment:
why `fn.system` over plenary.job ? Its blocking. plenary.job:start not

Reply:
I didn't notice a difference and wasn't getting blocked, so I changed it because I thought it looked cleaner. I can change it back.